velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -1499,4 +1504,39 @@ struct ToISO8601Function {
   }
 };
 
+template <typename T>
+struct TimestampWithTimezoneAtTimezoneFunction
```

Comment:
why not simply calling it `AtTimezoneFunction` to follow the pattern from other functions?

Reply:
Thanks for the review. This naming was leftover from when I had implementations for different function signatures, I glossed over it when refactoring. Cleaned them all up.